content/riak/kv/2.2.0/using/performance/open-files-limit.md

```diff
@@ -15,326 +15,278 @@ aliases:
 
-```config
+```/etc/ssh/sshd_config
 UseLogin yes
 ```
 
-6\. Restart the machine so that the limits to take effect and verify that
-the new limits are set with the following command:
+6\. Restart the machine so the limits take effect and verify that the new limits are set with the following command:
 
 ```bash
 ulimit -a
 ```
 
+{{% note %}}
+In the above examples, the open files limit is raised for all users of the system. The limit can be specified for the riak user only by substituting the
+two asterisks (`*`) in the examples with `riak`.
+{{% /note %}}
+
+
+## CentOS & Red Hat
+
+Start by checking the current open file limit values with:
+
+```bash
+ulimit -Hn # Hard limit
+ulimit -Sn # Soft limit
+```
+
+If you installed Riak KV from a binary package, you will need to the add the following settings to the /etc/security/limits.conf file for the `riak` user:
+
+```/etc/security/limits.conf
+riak soft nofile 65536
+riak hard nofile 200000
+```
+
+If you use initialization scripts to start Riak KV, you can create a /etc/default/riak file and add the following to specify a limit:
+
+```/etc/default/riak
+ulimit -n 200000
+```
+
+This file is automatically sourced from the initialization script, and the Riak KV process will inherit this setting. Since initialization scripts are always run as the root user, there’s no need to set limits in /etc/security/limits.conf.
+
 ### Enable PAM-Based Limits for CentOS and Red Hat
 
-1\. Edit `/etc/security/limits.conf` and append the following lines to
-the file:
+You can enable PAM-based user limits so that non-root users, such as the `riak` user, may specify a higher value for maximum open files.
 
-    <div><pre><code>&ast;              soft     nofile          200000
-    &ast;              hard     nofile          200000
-    </code></pre></div>
+For example, follow these steps to enable PAM-based limits for all users to allow a maximum of 200000 open files.
 
-2\. Save and close the file.
+1\. Edit /etc/pam.d/login and add the following line:
 
-3\. Restart the machine so that the limits to take effect and verify that
+```/etc/pam.d/login
+session    required   pam_limits.so
+```
+
+2\. Save and close /etc/pam.d/login
+
+3\. Edit /etc/security/limits.conf and append the following lines to the file:
+
+```/etc/security/limits.conf
+* soft nofile 65536
+* hard nofile 200000
+```
+
+4\. Save and close the /etc/security/limits.conf file.
+
+5\. Restart the machine so that the limits to take effect and verify that
 the new limits are set with the following command:
 
 ```bash
 ulimit -a
 ```
 
 {{% note %}}
-In the above examples, the
-open files limit is raised for all users of the system. If you prefer,
-the limit can be specified for the riak user only by substituting the
-two asterisks (`*`) in the examples with `riak<`.
+In the above examples, the open files limit is raised for all users of the system. The limit can be specified for the riak user only by substituting the
```

Comment:
Same thing here. I'm not fussed about whether or not it has the ` ` just so long as it has or doesn't have them consistently.